### PR TITLE
Add analogReference modes for the Arduino 33 BLE

### LINKED ISSUE #755: analogReference is missing information on nRF528x boards (Mbed OS)
I just bought an [Arduino Nano 33 BLE](https://store.arduino.cc/arduino-nano-33-ble). Looking at the [nRF52840 datasheet](https://content.arduino.cc/assets/Nano_BLE_MCU-nRF52840_PS_v1.1.pdf) (section *6.23.2 Reference voltage and gain settings*), it seems to have an internal 0.6 V reference. How do I select this with [`analogReference()`](https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/)?

### PR DESCRIPTION
**Pull request by slokhorst**
Found at https://github.com/arduino/ArduinoCore-mbed/blob/master/variants/ARDUINO_NANO33BLE/pins_arduino.h#L13

Fixes #755

### CODE REVIEW

File: Language/Functions/Analog IO/analogReference.adoc [hunk 1]
```diff
@@ -52,6 +52,14 @@ Arduino SAM Boards (Due)
 
 * AR_DEFAULT: the default analog reference of 3.3V. This is the only supported option for the Due.
 
+Arduino 33 BLE
```
[the remark this answers is not in the document]
Comment (slokhorst) [reply Language/Functions/Analog IO/analogReference.adoc]: Thanks for your comment! I've added the line you suggested.

**Comment by slokhorst** [comment]
Apparently analogReference support was just backported today to the "old" ArduinoCore-nRF528x-mbedos in arduino/ArduinoCore-nRF528x-mbedos#81, making it available in both platforms. I'm not sure when that will actually be available in a released version though.

**Comment by BallscrewBob** [comment]
Should these changes not also be mentioned in the reference section of the website as there has been a couple if mentions about the lack of documentation there by at least two users.

**Comment by slokhorst** [comment]
@BallscrewBob agreed. It took me a lot of searching to find out that there were two different platforms that support the 33 BLE.

It would be nice if there would be some kind of public roadmap for the two platforms. The only thing I could find was arduino/ArduinoCore-nRF528x-mbedos#81, which states:
> This should be the last maintenance release of this core; since this core is based on mbed 5.x branch all functions marked for deprecation are still there; arduino/ArduinoCore-mbed, on the other hand, contains all the new stuff and should replace this core as soon as considered stable.

@facchinm do you know more? If/when is ArduinoCore-mbed expected to replace ArduinoCore-nRF528x-mbedos? What feature differences are there between the two platforms? Is there already documentation available on this that we overlooked?

**Comment by per1234** [comment]
>Apparently analogReference support was just backported today to the "old" ArduinoCore-nRF528x-mbedos in arduino/ArduinoCore-nRF528x-mbedos#81, making it available in both platforms. I'm not sure when that will actually be available in a released version though.

Thanks for the clarification @slokhorst. We shouldn't mention that in the reference now, because as you say it hasn't yet been released and so isn't available to the readers of the Arduino Language Reference. When/if there is a release of  the "Arduino nRF5248x Boards (Mbed OS)" platform, I would welcome a PR to update this information accordingly.

> Should these changes not also be mentioned in the reference section of the website

@BallscrewBob this repository contains the content of the Arduino Language Reference:
https://www.arduino.cc/reference/en/

Arduino set up this really awesome system that makes it so that, after a pull request is merged in this repository, those changes will be published to the Arduino Language Reference via a completely automated system. This means we should soon see the content @slokhorst contributed here also added to the [`analogReference()` reference page](https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/) without any additional human effort required.

**Pull request merged**